### ISSUE TITLE
Move variables and functions out of environment 

### ISSUE THREAD

**Issue by cyrusae** [cyrusae/helpAR#2]
- [ ] Sweep all functions contributing to helpAR under helpAR itself
- [ ] Try sweeping the table generation under it as well 
- [ ] Test for hits to performance from (1) and (2), modify accordingly. But it would be nice for the only variable it generates in the environment, even as a notebook, to be the function itself.

**Comment by cyrusae** [comment]
Would benefit from being done after #1 so I don't have to go out of my way to get data to test on.

**Comment by cyrusae** [comment]
Superseded by #3

**Comment by cyrusae** [comment]
.
